### PR TITLE
Add landing message #1

### PR DESCRIPTION
**Pull request by YYH-coder-0813**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Welcome to Pontiony</title>
+    </head>
+    <body>
+        <h1>Welcome to Poionfy! Enjoy your stay</h1>
```
Comment (yhh00001) [inline index.html]: could u remove Enjoy ... and make welcome string to red

Comment (YYH-coder-0813) [reply index.html]: sure can do

**Review by yhh00001 — CHANGES_REQUESTED** [review]
Tq change it now

**Review by yhh00001 — APPROVED** [review]
well done

**Pull request merged**